### PR TITLE
OC-12933 Signature attestation is appearing when form is getting unsigned

### PR DESCRIPTION
**Pull request by jkeremian**
This PR contains fixes for the following stories.
OC-12933 Signature attestation is appearing when form is getting unsigned
OC-10309 When user removes and restores a form under a signed event or the event itself then event should become unsigned
OC-12926 If completed signed common event is removed and then restored, no sign button appears after form is restored

### CODE REVIEW

**Comment by svadla-oc** [comment]
👍

**Comment by svadla-oc** [comment]
@jkeremian can you resolve these conflicts please?

**Comment by jkeremian** [comment]
Merge conflict issue should be resolved now

File: src/main/java/core/org/akaza/openclinica/bean/extract/odm/ClinicalDataReportBean.java [hunk 1]
```diff
@@ -811,11 +811,11 @@ protected void addOneAuditLog(AuditLogBean audit, String currentIndent) {
                 }
                 if (o.length() > 0) {
                     xml.append(nls);
-                    xml.append(currentIndent + "                      OldValue=\"" + org.apache.commons.lang3.StringEscapeUtils.escapeXml11(o) + "\" ");
+                    xml.append(currentIndent + "                      OldValue=\"" + StringEscapeUtils.escapeXml(o) + "\" ");
                 }
                 if (n.length() > 0) {
                     xml.append(nls);
-                    xml.append(currentIndent + "                      NewValue=\"" + org.apache.commons.lang3.StringEscapeUtils.escapeXml11(n) + "\"");
+                    xml.append(currentIndent + "                      NewValue=\"" + StringEscapeUtils.escapeXml(n) + "\"");
```
Comment (svadla-oc) [inline src/main/java/core/org/akaza/openclinica/bean/extract/odm/ClinicalDataReportBean.java:818]: looks like you reverted Suthirr's changes when resolving merge conflicts. Can you change it back?

Comment (jkeremian) [reply src/main/java/core/org/akaza/openclinica/bean/extract/odm/ClinicalDataReportBean.java]: all set , reverted back

**Comment by svadla-oc** [comment]
👍

**Pull request merged**